jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard.js

```diff
@@ -466,11 +383,19 @@ define([
             var width = layout.width || computedLayout.width;
             var height = layout.height || computedLayout.height;
 
-            $cell
-                .attr('data-gs-x', layout.col)
-                .attr('data-gs-y', layout.row)
-                .attr('data-gs-width', width)
-                .attr('data-gs-height', height);
+            $cell.attr('data-gs-width', width)
+                 .attr('data-gs-height', height);
+
+            if (layout.hasOwnProperty('col') && layout.hasOwnProperty('row')) {
```

Comment:
Should just be able to do `if (layout.col && layout.row)`, no need for `hasOwnProperty`.